cypress/support/e2e.ts

```diff
@@ -14,7 +14,8 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import './commands'
-
+// eslint-disable-next-line import/extensions,import/no-unresolved
+import "./commands";
+import "cypress-file-upload";
```

Comment:
You are already importing `commands`, where you also import `cypress-file-upload`. See the [docs for the package](https://www.npmjs.com/package/cypress-file-upload#installation)